metal3-dev/local-cbo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -xe
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+LOGDIR=${SCRIPTDIR}/logs
+source $SCRIPTDIR/logging.sh
+
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/utils.sh
+
+if ! which kubebuilder 2>&1 >/dev/null ; then
```

Comment:
Is this checking a dependency that the CBO Makefile should handle?

Reply:
Not directly, I though at the beginning it could have been useful to ensure its presence during the early stage of development. As it stands right now, I think it could also be removed.